src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -57,6 +57,10 @@ public ReadOnlyPerson getPerson() {
         return toAdd;
     }
 
+    public boolean isMutating(){
```

Comment:
There should be an `@Override` to indicate that this child class is overriding its parent's class method.